### PR TITLE
feat: show total tokens in the asset list

### LINKED ISSUE #8006: [Task]: show total tokens in the asset list
### Task description

Currently only the available tokens are shown in the asset list
Add in the token info popup the possibility of seeing all tokens: the available tokens and the locked tokens (by conditions like timelock or expiration)

- Tile main value: total balance (except in mana, in mana it should be available mana)
- Token popup, create new balance section
    - Available: given by the sdk
    - Conditionally locked: total - available

### LINKED ISSUE #8229: [Task]: fix consistency between available mana
### Task description

The mana available in the wallet summary and that in the assetlist is not the same and it should be

### PR DESCRIPTION
**Pull request by evavirseda**
## Summary

> Closes #8006 and closes #8229
...

## Changelog

```
- Please write a list of changes
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/components/popups/TokenInformationPopup.svelte [hunk 1]
```diff
@@ -127,9 +140,16 @@
                         isCopyable
                     />
                 {/if}
+                <balance-wrapper class="flex flex-col bg-gray-50 dark:bg-gray-850 px-4 py-4 rounded-lg">
+                    <BalanceSummarySection
+                        titleKey="asset"
+                        subtitleKey="asset"
+                        amount={balanceSummary?.amount}
+                        subBreakdown={balanceSummary?.details}
```
Comment (cpl121) [inline packages/desktop/components/popups/TokenInformationPopup.svelte:148]: bad format when calculating base unit token
Captura de pantalla de 2024-03-25 17-31-40

File: packages/shared/components/tiles/AssetTile.svelte [hunk 2]
```diff
@@ -15,6 +16,7 @@
 
     $: marketPrice = asset ? getMarketPriceForAsset(asset) : undefined
     $: marketBalance = asset ? getMarketAmountFromAssetValue(asset.balance?.total, asset) : undefined
+    $: total = (asset?.id === MANA_ID ? asset?.balance?.available : asset?.balance?.total) ?? 0
```
Comment (cpl121) [inline packages/shared/components/tiles/AssetTile.svelte:19]: I don't understand this line 🥲

**Review by cpl121 — CHANGES_REQUESTED** [review]

**Review by cpl121 — APPROVED** [review]
🚀

File: packages/shared/components/BalanceSummarySection.svelte [hunk 3]
```diff
@@ -32,16 +37,28 @@
         }
     }
 
-    function handleAmount(isBaseToken: boolean, amount: number) {
-        return isBaseToken ? getAmount(amount) : getAmountMana(amount)
+    function handleAmount(isBaseToken: boolean = false, amount: number) {
```
Comment (marc2332) [inline packages/shared/components/BalanceSummarySection.svelte:40]: This should be receiving `asset` as function parameter, just like `isBaseToken` and `amount`

File: packages/shared/components/BalanceSummarySection.svelte [hunk 4]
```diff
@@ -20,7 +21,11 @@
     $: ({ baseCoin } = $selectedWalletAssets?.[$activeProfile?.network?.id] ?? {})
 
     function getAmount(amount: number): string {
-        return baseCoin?.metadata ? formatTokenAmountBestMatch(amount, baseCoin?.metadata) : ''
+        if (!asset) {
```
Comment (marc2332) [inline packages/shared/components/BalanceSummarySection.svelte:24]: same

File: packages/shared/components/BalanceSummarySection.svelte [hunk 5]
```diff
@@ -32,16 +37,28 @@
...
+            return isBaseToken ? getAmount(amount) : getAmountMana(amount)
+        } else {
+            return getAmount(amount)
+        }
     }
 
-    function handleCurrencyAmount(isBaseToken: boolean, amount: number) {
-        return isBaseToken ? getCurrencyAmount(amount) : getAmountMana(amount)
+    function handleCurrencyAmount(isBaseToken: boolean = false, amount: number) {
```
Comment (marc2332) [inline packages/shared/components/BalanceSummarySection.svelte:48]: same

File: packages/shared/components/BalanceSummarySection.svelte [hunk 6]
```diff
@@ -32,16 +37,28 @@
...
+        if (!asset) {
+            return isBaseToken ? getCurrencyAmount(amount) : getAmountMana(amount)
+        } else {
+            return getCurrencyAmount(amount)
+        }
     }
 
     function getCurrencyAmount(amount: number): string {
-        return baseCoin ? formatCurrency(getMarketAmountFromAssetValue(amount, baseCoin)) : ''
+        if (!asset) {
```
Comment (marc2332) [inline packages/shared/components/BalanceSummarySection.svelte:57]: same, and you might as well add `baseCoin`

**Review by marc2332 — CHANGES_REQUESTED** [review]

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
- base token is being displayed with mana
image
- remove the secondary value if the asset has no fiat value
- in token details popup, simplify the breakdown to be:
    - Total Balance                                amount
          - Available                                 amount
          - Conditionally Locked              amount

File: packages/shared/components/BalanceSummarySection.svelte (hunk 6 above)
Comment (marc2332) [reply packages/shared/components/BalanceSummarySection.svelte:57]: Now that I think about this one, I don't think it was worth adding, but it's fine

**Review by marc2332 — APPROVED** [review]

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**